schema.sql

```diff
@@ -1,20 +1,19 @@
 /* Database schema to keep the structure of entire database. */
+-- \c vet_clinic
 
-/* Created database */
-
-CREATE DATABASE vet_clinic;
-
-/* Access database */
-
-psql vet_clinic;
+CREATE TABLE animals (
+    id INT  PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name varchar(255) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    escape_attempts INT,
+    neutered BOOLEAN,   
+    weight_kg DECIMAL
+);
 
-/* Created table */
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES
+('Agumon','2020-02-03',0,true,'10.23'),
+('Gabumon','2018-11-15',2,true,'8'),
+('Pikachu', '2021-01-07',1,false, '15.04'),
+('Devimon', '2017-05-12', 5, true, '11');
```

Comment:
- [ ] Please remove these lines of code from this file.  Kindly note that Everything that populates tables with data should be saved in the data.sql file.

Reply:
Done, thanks.